avocado/core/__init__.py

```diff
@@ -195,6 +195,18 @@ def register_core_options():
                          default=[],
                          help_msg=help_msg)
 
+    help_msg = ('Selects the runner implementation from one of the '
+                'installed and active implementations.  You can run '
+                '"avocado plugins" and find the list of valid runners '
+                'under the "Plugins that run test suites on a job '
+                '(runners) section.  Defaults to "nrunner", which is '
+                'the new runner.  To use the conventional and traditional '
+                'runner, use "runner".')
+    stgs.register_option(section='run',
```

Comment:
this is great change, and I think that as a part of this commit should also be code refactoring. There are still places where the default value is created like [suite](https://github.com/avocado-framework/avocado/blob/master/avocado/core/suite.py#L259)

Reply:
> > Hi @clebergnu, the code LGTM, thanks. I would just to propose more refactoring of default runner variable inside code. So we will avoid any problems in the future. Also, it looks like there is some problem with tests.
> 
> I'm looking at the test failure... but I'm not sure I understand what you mean by "more refactoring of default runner variable inside code". Care to give an example?

Never mind, I see what you meant in the other comment.